README.md

```diff
@@ -80,6 +80,16 @@ print(task.result().measurement_counts)
 
 The code sample imports the Amazon Braket framework, then defines the device to use (the SV1 AWS simulator). The `s3_folder` statement defines the Amazon S3 bucket for the task result and the folder in the bucket to store the task result. This folder is created when you run the task. It then creates a Bell Pair circuit, executes the circuit on the simulator and prints the results of the job. This example can be found in `../examples/bell.py`.
 
+### Running multiple tasks at once
+
+Many quantum algorithms need to run multiple independent circuits, and running the circuits in parallel is much faster than running them one at a time. The following example shows how to run a batch of tasks:
```

Comment:
Need to distinguish between sims and QPUs:

Many quantum algorithms need to run multiple independent circuits. Submitting the circuits in parallel can be faster than submitting them one at a time. In particular, if you use simulator devices, parallel processing of your tasks provide a significant speed up.